Makefile

```diff
@@ -70,12 +70,18 @@ MKBIN_D = $(CXX) $(CXXFLAGS) $(FLAGS_D) $^ $(LDFLAGS) -o $@
 
 ################################################################################
 
+# Functions
+
+rwildcard=$(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2) $(filter $(subst *,%,$2),$d))
+
+################################################################################
+
```

Comment:
I think adding a new "segment" for just this function is a little jarring, considering the segment below it is much larger and contains many different kinds of statements.

If we are going to add more segments (as is also done in #4), we may want to rewrite this line:
`# The lines above the #-line can be altered to change the build behavior.`